smarts/core/controllers/__init__.py

```diff
@@ -84,13 +83,9 @@ def perform_action(
                 The state of a vehicle sensor as relates to vehicle sensors.
             action_space:
                 The action space of the provided action.
-            vehicle_type:
-                Vehicle type information about the given vehicle.
         """
         if action is None:
             return
-        if vehicle_type == "bus":
```

Comment:
Is this necessary? This was the only use of `vehicle_type` in this function so would a bus still be able to work properly?

Reply:
I doubt this necessary. I would also prefer to have fewer cases whenever possible.